src/cli/ionic-config.ts

```diff
@@ -0,0 +1,45 @@
+import { readJSON, writeJSON, mkdirp } from '@ionic/utils-fs';
+
+import * as os from 'os';
+import * as path from 'path';
+
+import { uuidv4 } from './telemetry/helpers';
+
+export const CONFIG_FILE = 'config.json';
+export const DEFAULT_CONFIG_DIRECTORY = path.resolve(os.homedir(), '.ionic', CONFIG_FILE);
+
+export interface SystemConfig {
+	"telemetry"?: boolean,
+	"telemetry.stencil"?: boolean,
+	"tokens.telemetry"?: string,
```

Comment:
For my own understanding, is this a series of strings with a comma (or something similar) as a delimiter?

Reply:
`tokens.telemetry` is the result of the uuid function that gets called on Line 26 - it's just a plain string with some dashes. Looks kind of like "xxxxxxx-xxxxx-xxxx" (that's not an accurate format, I'm on a walk)